iOS10-HW12-Alexandr-Ivanov/Sources/ViewController.swift

```diff
@@ -0,0 +1,151 @@
+        label.textColor = .white
+        label.textAlignment = .center
+        label.text = "WORK"
+        label.translatesAutoresizingMaskIntoConstraints = false
+
+        return label
+    }()
+
+    // MARK: - Life Cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupView()
+        setupHierarchy()
+        setupLayout()
+    }
+
+    // MARK: - Setups
+    private func setupView() {
+        view.backgroundColor = .black
+    }
+
+    private func setupHierarchy() {
+        view.addSubview(circularProgressBarView)
+        view.addSubview(timeLabel)
+        view.addSubview(startStopButton)
+        view.addSubview(titleLabel)
+    }
+
+    private func setupLayout() {
+        NSLayoutConstraint.activate([
+            circularProgressBarView.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            circularProgressBarView.centerYAnchor.constraint(equalTo: view.centerYAnchor),
+            timeLabel.leftAnchor.constraint(equalTo: view.leftAnchor, constant: 20),
+            timeLabel.rightAnchor.constraint(equalTo: view.rightAnchor, constant: -20),
+            timeLabel.centerYAnchor.constraint(equalTo: view.centerYAnchor),
+            startStopButton.heightAnchor.constraint(equalToConstant: 45),
+            startStopButton.widthAnchor.constraint(equalToConstant: 45),
+            startStopButton.centerYAnchor.constraint(equalTo: view.centerYAnchor, constant: view.bounds.height / 3.5),
+            startStopButton.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            titleLabel.leftAnchor.constraint(equalTo: view.leftAnchor, constant: 20),
+            titleLabel.rightAnchor.constraint(equalTo: view.rightAnchor, constant: -20),
+            titleLabel.centerYAnchor.constraint(equalTo: view.centerYAnchor, constant: -view.bounds.height / 3.5)
+        ])
+    }
+
+    // MARK: - Actions
+
+    @objc private func buttonTapped() {
+        isStarted.toggle()
+
+        switch isStarted {
+        case true:
+            circularProgressBarView.resumeAnimation()
+            timer = Timer.scheduledTimer(timeInterval: 0.01, target: self, selector: #selector(update), userInfo: nil, repeats: true)
+            startStopButton.tintColor = .systemOrange
+            startStopButton.setBackgroundImage(UIImage(systemName: "pause"), for: .normal)
+        default:
+            circularProgressBarView.pauseAnimation()
+            timer?.invalidate()
+            startStopButton.tintColor = .white
+            startStopButton.setBackgroundImage(UIImage(systemName: "play"), for: .normal)
+        }
+    }
+
+    @objc private func update() {
+        let formatter = DateFormatter()
+        formatter.dateFormat = "mm:ss"
+        let date = Date(timeIntervalSince1970: TimeInterval(runCount))
+        timeLabel.text = formatter.string(from: date)
+        let rounded = round(runCount * pow(10, 1)) / pow(10, 1)
+
+        if rounded == 25 && isWorkTime {
```

Comment:
```suggestion
        if rounded == 25 && isWorkTime {
```
Вот тут 25 это какой-то хардкод
Сделай под это константу чтобы можно было ее вверху где все данные менять и не лезть в код и везде исправлять